### PR TITLE
Bump gems

### PR DESCRIPTION
**Pull request by smith558**
CHANGELOG notes
- [x] **`simplecov`** - The release `0.18.x` is broken on cloud CIs, freezing to `< 0.18`
https://github.com/codeclimate/test-reporter/issues/418

MAJOR version updates CHANGELOG
- [x] **`rails`** - Bumped to 6.0.3.2 from 5.0.7.2
https://guides.rubyonrails.org/6_0_release_notes.html
https://guides.rubyonrails.org/5_2_release_notes.html
https://guides.rubyonrails.org/5_1_release_notes.html

- [x] **`i18n`** - Bumped to 1.8.5 from 0.9.5, breaking change fix (https://github.com/ruby-i18n/i18n/releases/tag/v1.1.0)
https://github.com/codeclimate/test-reporter/issues/418

- [x] **`method_source`** - Bumped to 1.0.0 from 0.9.2
https://github.com/banister/method_source/blob/master/CHANGELOG.md

- [x] **`sprockets`** - Bumped to 4.0.2 from 3.7.2
https://github.com/rails/sprockets/blob/master/CHANGELOG.md

- [x] **`sidekiq`** - Bumped to 6.1.1 from 5.2.9
https://github.com/mperham/sidekiq/blob/master/Changes.md

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
-require "active_job/railtie"
-require "active_record/railtie"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_view/railtie"
-require "action_cable/engine"
-require "action_cable/engine"
-require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require 'rails/all'
```
[the remark this answers is not in the document]
Comment (smith558) [reply config/application.rb:3]: okay

File: config/initializers/new_framework_defaults.rb [hunk 2]
```diff
@@ -6,18 +6,5 @@
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 
-# Enable per-form CSRF tokens. Previous versions had false.
```
[the remark this answers is not in the document]
Comment (smith558) [reply config/initializers/new_framework_defaults.rb:9]: To uz nebolo treba. Presiel som si tie defaults a nijako by nas nemali zasahovat, takze som zmazal (podla navodu), tym padom pouziva defaults s current version. Potom spustil testy.

File: config/environments/test.rb [hunk 3]
```diff
@@ -15,18 +17,23 @@
...
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    'Cache-Control' => 'public, max-age=3600'
+    'Cache-Control' => "public, max-age=#{1.hour.to_i}"
   }
 
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.cache_store = :null_store
```
[the remark this answers is not in the document]
Comment (smith558) [reply config/environments/test.rb:26]: z rails update task `app:update` (podla navodu), automaticky generovane

File: config/environments/production.rb [hunk 4]
```diff
@@ -78,9 +84,30 @@
...
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
-    config.logger = ActiveSupport::TaggedLogging.new(logger)
+    config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Inserts middleware to perform automatic connection switching.
```
[the remark this answers is not in the document]
Comment (smith558) [reply config/environments/production.rb:93]: Staging u nas vlastne kopiruje production, v konfiguracii, spravne?

File: config/environments/production.rb [hunk 5]
```diff
@@ -14,6 +14,10 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  config.require_master_key = true
```
[the remark this answers is not in the document]
Comment (smith558) [reply config/environments/production.rb:19]: @pavolzbell Pravda, zle som pochopil co to vlastne robi. Takze tuto funkcionalitu nechceme vyuzivat?

[the remark this answers is not in the document]
Comment (pavolzbell) [reply config/environments/production.rb:19]: nepouziva sa, tak nechceme

File: config/environments/test.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (pavolzbell) [reply config/environments/test.rb:26]: ok

File: config/environments/production.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (pavolzbell) [reply config/environments/production.rb:93]: myslim, ze v tomto pripade ano (pozri ci to tak je predtym ako to upravis)

File: config/initializers/new_framework_defaults.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (pavolzbell) [reply config/initializers/new_framework_defaults.rb:9]: ok

File: Gemfile [hunk 6]
```diff
@@ -3,15 +3,11 @@ source 'https://rubygems.org'
 ruby '2.6.6'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.0.2'
+gem 'rails'
 # Use postgresql as the database for Active Record
 gem 'pg', '~> 0.20'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.6'
```
[the remark this answers is not in the document]
Comment (smith558) [reply Gemfile]: vyzera to ze nie

File: config/application.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (pavolzbell) [reply config/application.rb]: tuto ale zakomentujme vsetko co sa napouziva

File: config/application.rb [hunk 7]
```diff
@@ -1,15 +1,13 @@
...
 require "active_record/railtie"
-require "action_controller/railtie"
+require "active_storage/engine"
+# require "action_controller/railtie"
 require "action_mailer/railtie"
-require "action_view/railtie"
-require "action_cable/engine"
-require "action_cable/engine"
+# require "action_view/railtie"
+# require "action_cable/engine"
```
Comment (smith558) [inline config/application.rb:10]: @pavolzbell tu Action Cable netreba na nic?

Comment (pavolzbell) [reply config/application.rb:10]: @smith558 podla mna nie

File: config/application.rb [hunk 8]
```diff
@@ -1,15 +1,13 @@
 require_relative 'boot'
 
-require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
+# require 'rails/all'
 require "active_job/railtie"
 require "active_record/railtie"
-require "action_controller/railtie"
+require "active_storage/engine"
```
Comment (pavolzbell) [inline config/application.rb:6]: @smith558 storage asi tiez nepouzivame

File: config/application.rb [hunk 9]
```diff
@@ -1,15 +1,13 @@
...
+# require "action_controller/railtie"
 require "action_mailer/railtie"
```
Comment (pavolzbell) [inline config/application.rb:8]: mailer asi tiez nepouzivame, ci?

**Pull request merged**